tensorboard/plugins/base_plugin.py

```diff
@@ -75,6 +103,21 @@ def is_active(self):
     """
     raise NotImplementedError()
 
+  def frontend_metadata(self):
+    """Defines how the plugin will be displayed on the frontend.
+
+    The base implementation returns a default value. Subclasses are
+    encouraged to override this method and replace any attributes on the
+    result.
+    """
+    return FrontendMetadata(
+        tab_name=None,
+        disable_reload=False,
```

Comment:
Can we change this to `enable_reload`? Negative name can read funny. I know that our existing flag name was disable_reload but this might be the time to change that.